src/parser/warrior/arms/CONFIG.js

```diff
@@ -25,6 +25,10 @@ export default {
         This spec needs a focused maintainer. If you want to give it a try, check <a href="https://github.com/WoWAnalyzer/WoWAnalyzer">GitHub</a> for more information.
       </Warning><br />
 
+      If you find any issues or have something you'd like to see added, open an issue on <a href="https://github.com/WoWAnalyzer/WoWAnalyzer/issues/new">GitHub</a>, contact us on <a href="https://discord.gg/AxphPxU">Discord</a> or DM us on Discord.<br /><br />
+
+      Make sure to check out the <a href="https://discordapp.com/invite/Skyhold">Warrior Class Discord</a> if you need more specific advice or a more detailed guide than the ones available on <a href="https://www.icy-veins.com/wow/arms-warrior-pve-dps-guide">Icy-Veins</a> and <a href="https://www.wowhead.com/arms-warrior-guide">wowhead</a>.<br /><br />
+
       <Warning>
         Hi <kbd>@Zerotorescue</kbd> here. I updated this spec to be compatible with BFA, but I'm no Arms expert. If there's anything wrong please @poke me on Discord and I'll fix it if I can.
       </Warning>
```

Comment:
You should probably cleanup this config so as to remove line 32->34 and 22->26 since it is now supported. If line 20 isn't relevant, change that aswell.